src/exoticatechnologies/util/StringUtils.java

```diff
@@ -38,6 +38,28 @@ public static String getString(String parent, String key) {
         }
     }
 
+    /**
+     * Method that iterates through the <i>list</i> and searches for <i>string</i> while ignoring case
+     * If such an element is found, it returns <b>true</b>, otherwise returns <b>false</b>. Also returns <b>false</b> if <i>string</i> is null
+     *
+     * @param list the list to search for
+     * @param string the string to look for
+     * @return whether the string was found in the list or not while ignoring case
+     */
+    public static boolean containsIgnoreCase(List<String> list, String string) {
```

Comment:
hmm... isn't there an exactly identically named method to this one in `Extensions.kt` on line 580? Oh wait, the other method is just an extension method on a list of strings, and this is a method that takes a list of strings and a string..... very well, i'll let it slide...

Reply:
another clarifying comment, resolving.